spec/features/actors_feature_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "Actors", type: :feature do
+
+  before do
+    scorsese = create(:director, name: "Martin Scorsese", dob: "November 17, 1942")
+    departed = create(:movie, title: "The Departed", year: 2006, duration: 151, director: scorsese)
```

Comment:
Rather than hardcoding multiple movies, we might be better served by a sequence:

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences